model/project_parser.go

```diff
@@ -614,10 +618,16 @@ func evaluateBuildVariants(tse *taskSelectorEvaluator, tgse *tagSelectorEvaluato
 		}
 
 		// check that display tasks contain real tasks that are not duplicated
-		bvTasks := make(map[string]string)          // map of all execution tasks
+		bvTasks := make(map[string]struct{})        // map of all execution tasks
```

Comment:
Is there a reason you change the value here (not that it really matters)

Reply:
Explained in person.